trade-finance-manager-api/src/v1/api.js

```diff
@@ -1198,14 +1198,21 @@ const findBankById = async (bankId) => {
   }
 };
 
+/**
+ * @typedef {object} GetBanksQuery
+ * @property {boolean | undefined} includeReportingYears - Whether or not to include the bank reporting years
+ */
+
```

Comment:
Documentation seems funny here, can't see `includeReportingYears` anywhere?

Reply:
That's because we just pass in the query supplied to the `getBanks` function. This type becomes necessary inside the `banks.controller.ts` file where it now requires that the supplied query parameter matches this type definition